includes/utils.php

```diff
@@ -566,4 +566,22 @@ public static function parse_custom_attributes( $attributes_string, $delimiter =
 
 		return $result;
 	}
+
+	public static function find_element_recursive( $elements, $id ) {
+		foreach ( $elements as $element ) {
+			if ( $id === $element['id'] ) {
+				return $element;
+			}
+
+			if ( ! empty( $element['elements'] ) ) {
+				$element = self::find_element_recursive( $element['elements'], $id );
+
+				if ( $element ) {
+					return $element;
+				}
```

Comment:
```suggestion
				return self::find_element_recursive( $element['elements'], $id );
```
It will give the same result